components/codeflare/codeflare.go

```diff
@@ -71,6 +73,29 @@ func (c *CodeFlare) GetComponentName() string {
 	return ComponentName
 }
 
+func (c *CodeFlare) UpdateStatus(in *status.ComponentsStatus) error {
+	codeFlareStatus, err := c.GetReleaseVersion(in, deploy.DefaultManifestPath, ComponentName)
+
+	if err != nil {
+		in.CodeFlare = &status.CodeFlareStatus{}
+		return err
+	}
+
+	in.CodeFlare = &status.CodeFlareStatus{
+		ComponentStatus: status.ComponentStatus{
```

Comment:
would it be possible to move whole creation of ComponentStatus in one separate function leaving differences to parameters and methods?

Reply:
@ykaliuta this is done